modules/apps/commerce/commerce-product-api/src/main/java/com/liferay/commerce/product/servlet/taglib/ui/CPDefinitionScreenNavigationConstants.java

```diff
@@ -12,7 +12,7 @@
  * details.
  */
 
-package com.liferay.commerce.product.definitions.web.servlet.taglib.ui;
+package com.liferay.commerce.product.servlet.taglib.ui;
```

Comment:
@andrea-ale-sbarra we should actually change this to `package com.liferay.commerce.product.constants` or `package com.liferay.commerce.product.servlet.taglib.ui.constants` would work too. Just got to have constants be the ending path.